spec/support/seeds/seeds_e2e.rb

```diff
@@ -227,6 +227,7 @@
     icon: reading_icon,
     display_only_when_signed_in: true,
     position: 0,
+    section: 0,
```

Comment:
Same suggestion as above, we may want to use symbols here to make this a bit more readable for the next person.